nuxeo-distribution/nuxeo-nxr-server/src/main/resources/templates/common-base/nxserver/config/kafka-config.xml.nxftl

```diff
@@ -8,20 +8,22 @@
       <producer>
         <property name="bootstrap.servers">${kafka.bootstrap.servers}</property>
         <property name="default.replication.factor">${kafka.default.replication.factor}</property>
-<#if "${kafka.ssl}" == "true">
-<#if "${kafka.sasl.enabled}" == "true">
+<#if ((kafka.sasl.enabled)!"false") == "true" || ((kafka.ssl)!"false") == "true">
```

Comment:
All variables are set in the common-base/nuxeo.defaults template.
So why do we have theses default (like !"false") at someplace and not other?